### PR TITLE
feat: Adds ability to set first segment part as next by clicking header

### PR DESCRIPTION
**Pull request by RasmusAlbrektsen**
This PR includes the ability of clicking on the segment header and thereby setting the first part of the segment as next part. By request of @LindvedKrvang :-)

### CODE REVIEW

**Review by LindvedKrvang — APPROVED** [review]

File: src/app/rundown/components/segment/segment.component.ts [hunk 1]
```diff
@@ -91,6 +94,11 @@ export class SegmentComponent implements OnChanges, OnDestroy {
     this.timeReference = timeSpendUntilActivePart + timeSpendInActivePart
   }
 
+  public setFirstPartAsNext(): void {
+    const firstPart: Part = this.segment.parts[0]
```
Comment (andr9528) [inline src/app/rundown/components/segment/segment.component.ts:98]: Technically a Segment can have 0 parts, thus this _could_ return undefined. 
Don't know if this is something that is handled elsewhere, or even if it should be handled on the frontend.

**Review by andr9528 — APPROVED** [review]
Only had one comment, which isn't blocking a merge in my mind, just something that might be worth noting.

File: src/app/rundown/components/segment/segment.component.ts (hunk 1 above)
Comment (RasmusAlbrektsen) [reply src/app/rundown/components/segment/segment.component.ts:102]: For now i've added handling for the first part not being set as next if it contains no pieces. Whether the handling should extend further into scenarios where we e.g. have a first part as invalid and then should skip to the one after, is not considered. But i don't know if that is even relevant at this point in time.

**Review by LindvedKrvang — APPROVED** [review]

File: src/app/rundown/components/segment/segment.component.ts (hunk 1 above)
Comment (KvelaGorrrrnio) [reply src/app/rundown/components/segment/segment.component.ts:102]: Perhaps it should search for the first valid part, if any, instead.

**Review by KvelaGorrrrnio — APPROVED** [review]

File: src/app/rundown/components/segment/segment.component.ts (hunk 1 above)
Comment (RasmusAlbrektsen) [reply src/app/rundown/components/segment/segment.component.ts]: I have changed it to find a valid part instead!

**Pull request merged**